packages/commonwealth/client/scripts/views/components/sidebar/AccountConnectionIndicator/AccountConnectionIndicator.tsx

```diff
@@ -29,6 +32,23 @@ const AccountConnectionIndicator = ({
             <div className="status-row">
               <div className={clsx('status-light', { connected })} />
               <CWIdentificationTag address={address} />
+              <CWTooltip
+                placement="top"
+                content="address copied!"
+                renderTrigger={(handleInteraction) => {
+                  return (
+                    <CopySimple
```

Comment:
this is not the way we use icons in the project. We should use CWIcon or CWIconButton and add phosphor icon to cw_icon_lookup